packages/core/test/tabs/tabsTests.tsx

```diff
@@ -113,7 +113,10 @@ describe("<Tabs>", () => {
                 <Tab id="third" title="Third" className={tabClassName} panel={<Panel title="third" />} />,
             </Tabs>,
         );
-        assert.lengthOf(wrapper.find(`.${tabClassName}`), 9);
```

Comment:
Test is not changed to accommodate component changes; result is same as before-- see https://github.com/palantir/blueprint/pull/6898